src/settings-tab.ts

```diff
@@ -149,6 +149,22 @@ export class CustomFramesSettingTab extends PluginSettingTab {
                         await this.plugin.saveSettings();
                     });
                 });
+                new Setting(content)
+                .setName("Additional JS")
+                .setDesc(createFragment(f => {
+                    f.createSpan({ text: "A snippet of additional JS that should be applied to this frame." });
```

Comment:
I think it would be more user-friendly to use the full word "JavaScript" here.

Reply:
Fixed!